### PR TITLE
docs(getting-started): Fix getting started commands

### PR DESCRIPTION
**Pull request by JeremyLARDENOIS**
Replace command that need to be root
Correct paths
Change port of docker compose for controller

### CODE REVIEW

File: docs/src/introduction/getting_started.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ cargo run --release --bin scheduler
 
 Start the controller in a terminal
 ```bash
-cargo run --release --bin controller
+sudo ./target/release/controller
```
Comment (mflagey) [inline docs/src/introduction/getting_started.md:48]: Its not necessary to be super user here

File: docs/src/introduction/getting_started.md [hunk 2]
```diff
@@ -45,12 +45,12 @@ cargo run --release --bin scheduler
...
 ```
 
 Start the worker in a terminal
 ```bash
-cargo run --release --bin worker
+sudo ./target/release/riklet
```
Comment (mflagey) [inline docs/src/introduction/getting_started.md:53]: same here

File: docs/src/introduction/getting_started.md (hunk 1 above)
Comment (JeremyLARDENOIS) [reply docs/src/introduction/getting_started.md:48]: It's true, I will replace so with DATABASE_LOCATION=~/.rik/data cargo run --release --bin controller as you suggested

File: docs/src/introduction/getting_started.md (hunk 2 above)
Comment (JeremyLARDENOIS) [reply docs/src/introduction/getting_started.md:53]: Nope: https://github.com/polyxia-org/rik/blob/3e9c25d1524dc44762b6c28a713ba8e858e6bcaa/riklet/src/main.rs#L13

File: docs/src/introduction/getting_started.md (hunk 1 above)
Comment (JeremyLARDENOIS) [reply docs/src/introduction/getting_started.md]: Done

**Review by mflagey — APPROVED** [review]

**Review by AlexandreBrg — APPROVED** [review]

**Review by jlabatut — APPROVED** [review]

**Pull request merged**